app/src/main/java/org/simple/clinic/widgets/Views.kt

```diff
@@ -109,3 +111,12 @@ fun View.topRelativeTo(superParent: ViewGroup): Int {
 
   return totalDistance
 }
+
+fun View.showToast(@StringRes stringResId: Int, longToast: Boolean = false, vararg formatArgs: Any) {
+  showToast(context.getString(stringResId, formatArgs), longToast)
+}
+
+fun View.showToast(msg: CharSequence, longToast: Boolean = false) {
+  val toastLength = if (longToast) Toast.LENGTH_LONG else Toast.LENGTH_SHORT
+  Toast.makeText(context, msg, toastLength).show()
+}
```

Comment:
Based on the rules used by Jake and other team members of android-ktx, I think this is a wrong usage of extension function. 

https://www.youtube.com/watch?v=st1XVfkDWqk

Reply:
Got it. The core ktx [package](https://developer.android.com/reference/kotlin/androidx/core/widget/package-summary#extension-functions-summary) has an extension method for toasting via the `Context`. I can either add the library or implement the same signature myself.

Since we already use `work-runtime-ktx`, I guess we can use the core ktx library as well. Thoughts?

Reply:
Let's add android-ktx